### ISSUE TITLE
It crushes with an error when null value provided 

### ISSUE THREAD

**Issue by hoqua** [diedsmiling/keys-diff#10]
Hello @diedsmiling !
Thanks for useful lib.

But have you tried to 

```
const keysDiff = require('keys-diff')

const foo = {
  foo: 'foo',
  bar: {
    baz: 'baz',
    qux: 'qux'
  }
};

const bar = {
  foo: 'foo',
  bar: null,
  corge: 'corge'
};

keysDiff(foo, bar);

```
it gives me 
`TypeError: Cannot convert undefined or null to object`

**Comment by diedsmiling** [comment]
Hi!
Thanks a lot for reporting this, will check it.

**Comment by diedsmiling** [comment]
Published a fix in 1.0.8: https://www.npmjs.com/package/keys-diff
